src/components/common/calendar/Calendar.js

```diff
@@ -0,0 +1,192 @@
+import React, { useState, useEffect } from 'react';
+import '../../../styles/calendar.less';
+import 'antd/dist/antd.css';
+import { Calendar, Modal, Badge, Button } from 'antd';
+import CalendarModal from './CalendarModal';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
+
+const initialValues = [
+  {
+    date: '15/03/2022',
+    type: 'warning',
+    content: 'This is warning event.',
+    details: 'Test information 1',
+    time: '12:00 PM',
+  },
+  {
+    date: '15/03/2022',
+    type: 'success',
+    content: 'This is usual event.',
+    details: 'Test information 2',
+    time: '12:00 PM',
+  },
+  {
+    date: '16/03/2022',
+    type: 'error',
+    content: 'This is error event 1.',
+    details: 'Test information 3',
+    time: '12:00 PM',
+  },
+  {
+    date: '16/03/2022',
+    type: 'error',
+    content: 'This is error event 2.',
+    details: 'Test information 4',
+    time: '12:00 PM',
+  },
+  {
+    date: '16/03/2022',
+    type: 'error',
+    content: 'This is error event 3.',
+    details: 'Test information 5',
+    time: '12:00 PM',
+  },
+  {
+    date: '12/04/2022',
+    type: 'success',
+    content: 'This is usual event1.',
+    details: 'Test information 6',
+    time: '12:00 PM',
+  },
+  {
+    date: '12/04/2022',
+    type: 'success',
+    content: 'This is usual event2.',
+    details: 'Test information 7',
+    time: '12:00 PM',
+  },
+];
+
+function CalendarApp() {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isScheduleModalVisible, setIsScheduleModalVisible] = useState(false);
+  const [event, setEvent] = useState(null);
+  const [eventsArr, setEventsArr] = useState(initialValues);
+
+  // ****** will update with endpoint once db endpoint is built ******
```

Comment:
This commented code is the correct code but endpoint isn't built yet. To be uncommented when the endpoint is finished.